costmap_common:
    static_map: false
    rolling_window: true
    resolution: 0.05
    obstacle_range: 3.0
    raytrace_range: 4.0
    footprint: [[-0.02, -0.02], [0.02, -0.02], [0.02, 0.02], [-0.02, 0.02]]
    inflation_radius: 0.1
    observation_sources: lidar_sensor
    lidar_sensor: 
        sensor_frame: laser
        data_type: LaserScan
        topic: /scan
        marking: true
        clearing: true
